src/main/scala/org/bblfsh/client/v2/JNode.scala

```diff
@@ -0,0 +1,11 @@
+package org.bblfsh.client.v2
+
+sealed abstract class JNode
```

Comment:
Can you please add a comment to say what this is for, even if only briefly?

Reply:
addressed in 8162713fa88a425510391838a9f51c9bb83c5466